docs/docs/integrations/llms/azure_openai.ipynb

```diff
@@ -112,7 +112,7 @@
     "\n",
     "os.environ[\"OPENAI_API_TYPE\"] = \"azure\"\n",
     "os.environ[\"OPENAI_API_VERSION\"] = \"2023-05-15\"\n",
-    "os.environ[\"OPENAI_API_BASE\"] = \"...\"\n",
+    "os.environ[\"AZURE_ENDPOINT\"] = \"...\"\n",
```

Comment:
I believe the variables is called `AZURE_OPENAI_ENDPOINT` ([source](https://github.com/langchain-ai/langchain/blob/47b1b7092dac56794c8799d6af149fa00612086e/libs/partners/openai/langchain_openai/llms/azure.py#L36))

Reply:
Yes you are correct.